insights-client.spec

```diff
@@ -67,6 +77,16 @@ if [ -d %{_sysconfdir}/motd.d ]; then
     fi
 fi
 
+%post ros
+echo
+echo "Removing custom PCP configuration required for Resource Optimization service!"
+echo
```

Comment:
I think it's discouraged to echo output during scriptlet execution. Do you have any example precedent that shows that echoing  output to stdout is accepted behavior?

Reply:
I don't have any example, however I was neither able spot any guide/doc stating `echo` is discouraged/not allowed. I can remove that, though I wanted to have something for users to know what is happening.